assets/src/edit-story/components/library/panes/text/textSets/textSet.js

```diff
@@ -49,9 +49,12 @@ const TextSetItem = styled.div`
   background-color: ${({ theme }) =>
     theme.colors.interactiveBg.secondaryNormal};
   border-radius: ${({ theme }) => theme.borders.radius.small};
-  cursor: default;
+  cursor: pointer;
+  transition: background-color 0.3s;
```

Comment:
if ya want, `BUTTON_TRANSITION_TIMING` is available for import and would align this transition with other buttons, even though these don't look the same.